app/controllers/internal/AccountTypeController.scala

```diff
@@ -0,0 +1,111 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.internal
+
+import connector.FrontendAuthConnector
+import controllers.TarRules
+import controllers.internal.AccountType.AccountType
+import engine.RuleEngine
+import model.Locations._
+import model._
+import play.api.libs.json.{Json, Reads, Writes}
+import play.api.mvc.{Action, AnyContent, Request}
+import play.api.{Logger, LoggerLike}
+import services.{ThrottlingService, TwoStepVerification}
+import uk.gov.hmrc.play.frontend.auth.Actions
+import uk.gov.hmrc.play.frontend.controller.FrontendController
+import utils.EnumJson._
+
+object AccountType extends Enumeration {
+  type AccountType = Value
+  val Individual, Organisation = Value
+}
+
+case class AccountTypeResponse(`type`: AccountType)
+
+object AccountTypeResponse {
+  implicit val accountTypeReads = enumFormat(AccountType)
+  implicit val writes: Writes[AccountTypeResponse] = Json.writes[AccountTypeResponse]
+  implicit val reads: Reads[AccountTypeResponse] = Json.reads[AccountTypeResponse]
+}
+
+object AccountTypeController extends AccountTypeController {
+  override protected def authConnector = FrontendAuthConnector
+
+  override val defaultAccountType = AccountType.Organisation
+
+  override val ruleEngine = TarRules
+
+  override val logger: LoggerLike = Logger
+
+  override val throttlingService = ThrottlingService
+
+  override val twoStepVerification = TwoStepVerification
+
+  override def createAuditContext() = AuditContext()
+
+  // TODO: default location should be moved out from the controller. It belongs to the rule engine
+  override val defaultLocation = BusinessTaxAccount
+}
+
+trait AccountTypeController extends FrontendController with Actions {
+  def defaultAccountType: AccountType.AccountType
+
+  def logger: LoggerLike
+
+  def ruleEngine: RuleEngine
+
+  def throttlingService: ThrottlingService
+
+  def twoStepVerification: TwoStepVerification
+
+  def defaultLocation: Location
+
+  /**
+    *
+    * For the moment no audit event will be sent, but evaluation of the rules require the audit context
+    * so it is not removed until it is confirmed. We will most probably need auditing here too, but with a different type.
+    *
+    */
+  def createAuditContext(): TAuditContext
+
+  def accountTypeForCredId(credId: String) = Action.async { implicit request =>
+    val ruleContext = RuleContext(Some(credId))
+    val auditContext = createAuditContext()
+    // TODO calculate final destination should be refactor to return business type on a deeper layer (as opposed to process the destination)
+    calculateFinalDestination(ruleContext, auditContext) map {
+      case Locations.PersonalTaxAccount => Ok(Json.toJson(AccountTypeResponse(AccountType.Individual)))
```

Comment:
As discussed the rules can be _user type based_ and the location can be derived from the user type.